### PR TITLE
Fix duplicate link. Add note about inclusive language.

### PR DESCRIPTION
**Pull request by aphillips**
- Link checker flagged a duplicate link, fixed here.
- Pubrules warned about inclusive langauge use of the words "he", "his",
  and "her". I performed a complete search of the document. The terms
  "his" and "her" appear in the example of Hispanic names in section
  9.3.4. The term "he" also appears there. However, "he" appears
    throughout the document in the form of a language tag.

    To address, I added an HTML comment (not visible in the text) to
    remind ourselves in the future. We might consider providing an
    in-text note about inclusivity.

***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/bp-i18n-specdev/pull/110.html" title="Last updated on Jul 25, 2023, 2:56 PM UTC (b26c991)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/110/33c87af...aphillips:b26c991.html" title="Last updated on Jul 25, 2023, 2:56 PM UTC (b26c991)">Diff</a>

### CODE REVIEW

**Review by xfq — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -4,6 +4,13 @@
 	<meta charset="utf-8"/>
     <title>Internationalization Best Practices for Spec Developers</title>
 
+    <!--
+      Per the W3C Manual of Style, this document avoids gender-specific pronouns.
+      In this document, the terms "he", "his", and "her" appear in an example of gender-specific
+      cultural norms (related to personal name handling in applications), which represents a
+      well-considered expection to this rule. The word-token "he"
```
Comment (r12a) [inline index.html]: expection -> exception

File: index.html [hunk 2]
```diff
@@ -36,7 +43,7 @@
       
     </script>
     <!-- local styles for this document -->
-	<link rel="stylesheet" href="local.css">
+	<link rel="stylesheet" data-import href="local.css">
```
Comment (r12a) [inline index.html]: For this to work, i think you need to include the following in respecConfig:

````
	postProcess: [
		async function importStyleSheet() { 
			const elems = document.querySelectorAll(`link[rel='stylesheet'][data-import]`)
			await Promise.all(
				[...elems].map(async link => {
					const text = await fetch(link.href).then(r => r.text())
					const style = document.createElement("style")
					style.textContent = text
					link.replaceWith(style)
					})
				)
			}
		],
````

File: index.html (hunk 1 above)
Comment (aphillips) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (aphillips) [reply index.html]: fixed

**Comment by aphillips** [comment]
@r12a Please have a look. I made the changes we just discussed here _first_, since this is supposed to be the source for the stuff in tr-design (and it impacts the linking stuff we also discussed).

**Pull request merged**